plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -7873,10 +7871,26 @@ public void testCreateAvroTableWithSchemaUrl()
             throws Exception
     {
         String tableName = "test_create_avro_table_with_schema_url";
-        File schemaFile = createAvroSchemaFile();
+        TrinoFileSystem fileSystem = getTrinoFileSystem();
+        Location tempDir = Location.of("local:///temp_" + UUID.randomUUID());
+        fileSystem.createDirectory(tempDir);
+        String schema = """
```

Comment:
Id prefer to not inline the schema if possible. Im assuming it's because you want the test or class specific file system to handle everything?

Reply:
> Id prefer to not inline the schema if possible.

There is a single duplication of `createAvroSchemaFile`.
I find this repetition easy to digest for the reader & maintainer.

> Im assuming it's because you want the test or class specific file system to handle everything?
The query runner is based on `local://` file system locations and this change is in line with the changes from https://github.com/trinodb/trino/commit/ed67a55284372bb15ea21fae4257bf5dd78e055b